bin/install-jetpack.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
```

Comment:
I'm not convinced about the name `install-jetpack` , perhaps something like `run-jetpack-install` would be bettter.